yarn.lock

```diff
@@ -16433,7 +16433,7 @@ react-popper-tooltip@^2.11.0, react-popper-tooltip@^2.8.3:
     "@babel/runtime" "^7.9.2"
     react-popper "^1.3.7"
 
-react-popper@^1.3.7, react-popper@^2.2.1, react-popper@^2.2.3:
+react-popper@^1.3.7, react-popper@^2.2.1, react-popper@^2.2.4:
   version "2.2.3"
```

Comment:
Seems to me that the version isn't upgraded properly.
Tried to do a `yarn workspaces @lightspeed/flame add react-popper` and a `yarn workspaces @lightspeed/flame upgrade-interactive` both commands failed. Then I just manually upped the version in the package.json and did a `yarn`